src/com/google/javascript/jscomp/CompilationLevel.java

```diff
@@ -163,7 +163,7 @@ private static void applyFullCompilationOptions(CompilerOptions options) {
     options.setRemoveAbstractMethods(true);
     options.setRemoveSuperMethods(true);
     options.setReserveRawExports(true);
-    options.setRenamingPolicy(VariableRenamingPolicy.ALL, PropertyRenamingPolicy.ALL_UNQUOTED);
+    options.setRenamingPolicy(VariableRenamingPolicy.ALL, PropertyRenamingPolicy.OFF);
```

Comment:
@concavelenz Wondered if that was the case. No worries.

We'd be very surprised if turning off property renaming was safe in this case - because property removal can still happen. So can other breaking optimizations. In general, if it's not safe to rename, then most of the other property optimizations will have similar problems.